gui/util/range_slider.py

```diff
@@ -42,9 +42,9 @@ def __init__(self, slider_range, values, parent=None):
         else:
             self.setRange([0.0, 1.0, 0.01])
         if values:
-            self.values = values
+            self.setValues(values)
         else:
-            self.values = [0.3, 0.6]
+            self.setValues([0.3, 0.6])
```

Comment:
@kne42 can you elaborate on why this change is needed?

Reply:
@jni This was using properties before, which is a Pythonic implementation. To create a "truly" Qt-style class, we should be using get and set functions.

Reply:
I'm vaguely ok with a Pythonic "falsely" Qt class, as long as it works... ;) But I am not going to die on this particular hill. =P